api/src/main/java/org/openmrs/module/serialization/xstream/XStreamSerializer.java

```diff
@@ -159,6 +160,7 @@ protected MapperWrapper wrapMapper(MapperWrapper next) {
 		xstream.registerConverter(new CustomDynamicProxyConverter(), XStream.PRIORITY_VERY_HIGH);
 		// set our own defined marshalling strategy so that we can build references for cglib
 		xstream.setMarshallingStrategy(new CustomReferenceByIdMarshallingStrategy());
+		xstream.addPermission(AnyTypePermission.ANY);
```

Comment:
I think we shall not need the above after reverting to version `1.4.17` of xstream.

Reply:
I made the changes